src/me/danielschaefer/sensorwrangler/javafx/MainWindow.kt

```diff
@@ -351,8 +351,8 @@ class MainWindow(private val primaryStage: Stage, private val wrangler: SensorWr
 
                             currentText.text = "${(latestDataPoint?.value ?: 0.0)}${axis.unit.unitAppendix}"
                             // TODO: Is probably very inefficient for large measurements
-                            minText.text = filteredValues.min().toString()
-                            maxText.text = filteredValues.max().toString()
+                            minText.text = filteredValues.minOrNull().toString()
+                            maxText.text = filteredValues.maxOrNull().toString()
```

Comment:
Hmm, this might turn into a not so nice value.
Should do something like in line 352 to make sure we show a nice text when it's `null`.

Reply:
Done